ReportUtilityBills/measure.rb

```diff
@@ -298,54 +301,42 @@ def report_runperiod_output_results(runner, utility_bills, bill_scenario_name)
 
       if fuel_type == FT::Elec
-        if args[:electricity_bill_type] == 'Simple'
+        if bill_scenario.elec_tariff_filepath.nil?
           rate.fixedmonthlycharge = bill_scenario.elec_fixed_charge
           rate.flatratebuy = bill_scenario.elec_marginal_rate
-        elsif args[:electricity_bill_type] == 'Detailed'
-          if args[:electricity_utility_rate_type].get != 'Autoselect OpenEI'
-            if args[:electricity_utility_rate_type].get == 'User-Specified'
-              path = args[:electricity_utility_rate_user_specified].get
-
-              hpxml_path = @model.getBuilding.additionalProperties.getFeatureAsString('hpxml_path').get
-              filepath = FilePath.check_path(path,
-                                             File.dirname(hpxml_path),
-                                             'Tariff File')
-            else # sample rates
-              custom_rates_folder = File.join(File.dirname(__FILE__), 'resources/Data/CustomRates')
-              custom_rate_file = "#{args[:electricity_utility_rate_type].get}.json"
-              filepath = File.join(custom_rates_folder, custom_rate_file)
-            end
+        else
+          require 'json'
 
-            fileread = File.read(filepath)
-            tariff = JSON.parse(fileread, symbolize_names: true)
-            tariff = tariff[:items][0]
+          filepath = FilePath.check_path(bill_scenario.elec_tariff_filepath,
+                                         File.dirname(hpxml_path),
+                                         'Tariff File')
 
-            if tariff.keys.include?(:realtimepricing)
-              rate.fixedmonthlycharge = tariff[:fixedmonthlycharge] if tariff.keys.include?(:fixedmonthlycharge)
-              rate.realtimeprice = tariff[:realtimepricing].split(',').map { |v| Float(v) }
+          tariff = JSON.parse(File.read(filepath), symbolize_names: true)
+          tariff = tariff[:items][0]
 
-            else
-              rate.fixedmonthlycharge = tariff[:fixedmonthlycharge] if tariff.keys.include?(:fixedmonthlycharge)
-              rate.flatratebuy = tariff[:flatratebuy] if tariff.keys.include?(:flatratebuy)
+          if tariff.keys.include?(:realtimepricing)
+            rate.fixedmonthlycharge = tariff[:fixedmonthlycharge] if tariff.keys.include?(:fixedmonthlycharge)
+            rate.realtimeprice = tariff[:realtimepricing]
 
-              rate.energyratestructure = tariff[:energyratestructure] if tariff.keys.include?(:energyratestructure)
-              rate.energyweekdayschedule = tariff[:energyweekdayschedule] if tariff.keys.include?(:energyweekdayschedule)
-              rate.energyweekendschedule = tariff[:energyweekendschedule] if tariff.keys.include?(:energyweekendschedule)
+          else
+            rate.fixedmonthlycharge = tariff[:fixedmonthlycharge] if tariff.keys.include?(:fixedmonthlycharge)
+            rate.fixedmonthlycharge = tariff[:fixedchargefirstmeter] if tariff.keys.include?(:fixedchargefirstmeter)
```

Comment:
I'm confused, why do you look for two different fields? The [documentation](https://openei.org/services/doc/rest/util_rates/?version=3#response-fields) only lists `fixedmonthlycharge`, what is this other one?

Reply:
I saw `fixedchargefirstmeter` in at least one tariff file.

Reply:
Hmm perhaps usurdb.csv download is version 1? For example, column `fixedchargefirstmeter` exists and is documented. However, version 3's `fixedmonthlycharge` does not exist as a column.

Reply:
If we need to support `fixedchargefirstmeter`, the code above is definitely not correct. It's always being treated as a *monthly* fixed charge, but there is a separate `fixedchargeunits` which says whether it's daily/monthly/annual.

Reply:
I updated the `process_usurdb` method to map to `fixedmonthlycharge`, `minmonthlycharge`, and `annualmincharge` fields based on values in `fixedchargeunits` and `minchargeunits`.

Reply:
Sounds good. Do our calculations operate on all three (fixedmonthlycharge, minmonthlycharge, and annualmincharge) of those fields? BEopt has inputs for the first two, but not the latter... maybe I should add it.

Reply:
I think annualmincharge is used for "California-style calculations". See the "TODO" items in `util.rb`. I posted this as a question at the top of this PR.